spec/requests/errors_spec.rb

```diff
@@ -17,6 +17,21 @@
     end
   end
 
+  describe "bypassing user access checks" do
```

Comment:
These tests look good 🎉 But this doesn't feel like quite the right home for them. I would have suggested putting it in a DocumentsController spec but it turns out there isn't one!

I'd tentatively suggest creating a `spec/controllers` folder and testing it there, but @kevindew might have other thoughts...